### ISSUE TITLE
something issue about training

### ISSUE THREAD

**Issue by EggGDX** [FelixHertlein/illtrtemplate-model#5]
I am very interested in your research, so I downloaded 1/4 of the inv3d dataset for testing experiments. Unfortunately, the loss did not converge during training. Checking the output results, I found that the model's outputs are usually just a pure white image. Have you encountered this kind of issue? Do you have any training suggestions?
![image](https://github.com/user-attachments/assets/ac643b21-56d2-4a61-bfd5-d32b032976a2)

**Comment by FelixHertlein** [comment]
Thank you for your interest in our work!

No, I have not encountered this kind of issue. 
Did you try the following?

1. Check the training loss:
    - Is the loss value numeric or Inf/NaN?
    - Is the loss value reasonably high?
    - Does the loss decrease over time during training?

2. Check the inputs to the model directly before the forward and backward pass: [here](https://github.com/FelixHertlein/illtrtemplate-model/blob/b3ee560831f1fd39a2d941d1abc6b3687f5fe1ca/src/inv3d_illuminator/models/illtr/illtr_template.py#L92)
    - Inspect the three images (image_patch, albedo_patch, template). Do they look correct?
    - Are the values of the images scaled to 0 - 1?

3. Is the problem you data visualization code? Did you rescale the output of the model back to 0 - 255?
